src/app/core/services/github.service.ts

```diff
@@ -98,6 +98,15 @@ export class GithubService {
     );
   }
 
+  /**
+   * Creates a label in the current repository.
+   * @param labelName - name of new label.
+   * @param labelColour - colour of new label.
+   */
+  createLabel(labelName: string, labelColour: string): void {
+    octokit.issues.createLabel({owner: ORG_NAME, repo: REPO, name: labelName, color: labelColour});
```

Comment:
Lets be consistent here `color: labelColour` , color or colour. stick to one

Reply:
I'll keep it as color since its the API's requirement.